Script Analysis Examples/Dataspace Analysis/DataSpace_Default_Analysis.ipynb

```diff
@@ -419,40 +391,10 @@
     "#### Sample Output format\n",
     "\n",
     "```json\n",
-    "[\n",
-    "  {\n",
-    "    \"plot_label\": \"Input Voltage\",\n",
-    "    \"data\": [\n",
-    "      { \"id\": \"min\", \"result\": 1 },\n",
-    "      {\n",
-    "        \"id\": \"moving_mean\",\n",
-    "        \"x\": [1, 2, 3, 4, 5, 6],\n",
-    "        \"y\": [\"nan\", \"nan\", \"nan\", \"nan\", 3, 4]\n",
-    "      }\n",
-    "    ]\n",
-    "  },\n",
-    "  {\n",
-    "    \"plot_label\": \"Input Current\",\n",
-    "    \"data\": [\n",
-    "      { \"id\": \"min\", \"result\": 1 },\n",
-    "      {\n",
-    "        \"id\": \"moving_mean\",\n",
-    "        \"x\": [1, 2, 3, 4, 5, 6],\n",
-    "        \"y\": [\"nan\", \"nan\", \"nan\", \"nan\", 3, 4]\n",
-    "      }\n",
-    "    ]\n",
-    "  }\n",
-    "]\n",
-    "\n",
-    "```\n",
-    "where,\n",
-    "\n",
-    "- `plot_label` - Name of the channel.\n",
-    "- `data` - Analysis data of the channel.\n",
-    "- `id` - Unique ID of the analysis.\n",
-    "- `result` - Result of the analysis. This will apply to the scalar analysis type.\n",
-    "- `x` - Array of x-axis values of a data trace.\n",
-    "- `y` - Array of y-axis values of the analysis results. This will apply to the vector analysis type."
+    "{\n",
+    "    artifact_id: \"ec25561d-6509-49e5-9a78-30e9752733fe\"\n",
```

Comment:
It will be good to explain what this artifact means.

Reply:
Updated